core/src/main/web/app/controlpanel/controlpanel-directive.js

```diff
@@ -60,6 +63,17 @@
           return bkMenuPluginManager.getMenus();
         };
 
+        if (bkUtils.isElectron){
+          window.addEventListener('focus', function() {
```

Comment:
since the app is a single page app (for the most part) until there is a real page reload this will leak